README.md

```diff
@@ -15,3 +21,12 @@ automatically on edits:
 ```
 make serve
 ```
+
+## Team lists
+
+To update the teams, use the `team_query.py` from the theme. 
+
+```
+python team_query.py --org=scipy --team=scipy-core-team --title=Maintainers > maintainers.md
```

Comment:
Where should maintainers.md go?  Might be easier to add a Makefile target for this.

Need to explain that they need a GH_TOKEN variable exported and [where to get that token from](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).  It needs `read:org` permissions.